indico/modules/events/editing/client/js/components/Timeline/index.jsx

```diff
@@ -45,6 +45,7 @@ export default function Timeline() {
         state={lastState}
         submitter={timelineBlocks[0].submitter}
         eventId={eventId}
+        reviewConditionsOk={details.reviewConditionsOk}
```

Comment:
Maybe we could name this `validReviewConditions`? Seems more appropriate.

Reply:
`reviewConditionsValid` is better IMO. But thanks for suggesting a better name; I also wasn't a huge fan of the `Ok` but couldn't think of something better on the fly